### PR TITLE
Fixe #315 for master branch.

### PR DESCRIPTION
**Pull request by McNopper**
Fixed total count of VkDescriptorType in vkCreateDescriptorPoolhttps://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers/pull/315

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -4736,7 +4736,11 @@ static bool ValidateCmdBufImageLayouts(VkCommandBuffer cmdBuffer) {
         } else {
             if (cb_image_data.second.initialLayout == VK_IMAGE_LAYOUT_UNDEFINED) {
                 // TODO: Set memory invalid which is in mem_tracker currently
-            } else if (imageLayout != cb_image_data.second.initialLayout) {
+            } else if (imageLayout == VK_IMAGE_LAYOUT_UNDEFINED || imageLayout == VK_IMAGE_LAYOUT_PREINITIALIZED) {
+				// Filter out the transition from above old layouts:
```
Comment (lentinem) [inline layers/core_validation.cpp]: Please do not use tabs.

Comment (McNopper) [inline layers/core_validation.cpp]: Hmm, just checked my editors. All should to spaces instead of tabs. Maybe a copy and paste thing.

**Comment by McNopper** [comment]
Okay, rechecked with Notepad++. Seems Eclipse sometimes does not do the replace properly.

**Comment by McNopper** [comment]
#315 still valid and ready to be integrated.

**Comment by tobine** [comment]
Merged #315 and this PR appears to be duplicate so closing this. I separately pushed fix for comment indentation.

**Pull request closed without merging**